Sources/Swift/Tools/SentryViewPhotographer.swift

```diff
@@ -1,16 +1,12 @@
 #if canImport(UIKit) && !SENTRY_NO_UIKIT
 #if os(iOS) || os(tvOS)
 
-@_implementationOnly import _SentryPrivate
 import CoreGraphics
 import Foundation
 import UIKit
 
-@available(iOS, introduced: 16.0)
-@available(tvOS, introduced: 16.0)
 @objcMembers
-class SentryViewPhotographer: NSObject {
-    
+class SentryViewPhotographer: NSObject, SentryViewScreenshotProvider {
```

Comment:
`m`: This class has a low test coverage. Does it make sense to add some tests for these changes in this PR?

Reply:
This bugs me too, but this class relies too much on UI and drawing, testing the Image results will most likely be flaky.

I will try to extract some code from here that can be tested.